controller/ctrlUsers.js

```diff
@@ -180,11 +184,52 @@ const registerUser = async (req, res, next) => {
   }
 };
 
+const updateAvatar = async (req, res, next) => {
+  const { _id } = req.user;
+  const avatarURL = `/avatars/av_${_id}.png`;
+  Jimp.read(`tmp/${req.file.filename}`)
```

Comment:
Chcialem napisac: "to zdecydowanie powinno byc `jimp` (z malej)". Ale... Przyklady w dokumentacji jimp sa dokladnie takie. Fuckin' hell 🤦‍♂️

Reply:
Też nad tym myślałem, cóż, tak chcą :D